src/nycdb/dataset.py

```diff
@@ -64,9 +65,32 @@ def index(self):
         else:
             logging.debug('no index files exist for this dataset')
 
+    def _get_transformed_dataset_rows_using_module(self, schema, dataset_transformations_module):
+        rows_func = getattr(dataset_transformations_module, schema['table_name'], None)
+        if rows_func:
+            return rows_func(self)
+        rows_func = getattr(dataset_transformations_module, self.name, None)
+        if rows_func:
+            return rows_func(self, schema)
+        return None
+
+    def _get_transformed_dataset_rows(self, schema):
+        for plugin in plugins.iter_plugins():
+            dt_module = plugin.import_submodule('dataset_transformations')
+            rows = self._get_transformed_dataset_rows_using_module(schema, dt_module)
+            if rows is not None:
+                return rows
+
+        table_name = schema['table_name']
+        raise Exception(
+            f"Unable to find transformer for dataset {self.name}, "
```

Comment:
I like this name. We think we should rename the module `dataset_transformations` to `transformers`

Reply:
Ah nice, let's do that!